### ISSUE TITLE
Clearsigned message verification isn't working

### ISSUE THREAD

**Issue by petertodd** [keybase/keybase-issues#1755]
I tried to verify the following message via https://keybase.io/verify, and got the error message "Error: you cannot process a message that was encrypted for someone else."

    -----BEGIN PGP SIGNED MESSAGE-----
    Hash: SHA256
    
    hello! whose signed this message?
    -----BEGIN PGP SIGNATURE-----
    
    iQGrBAEBCACVBQJV5H8tXhSAAAAAABUAQGJsb2NraGFzaEBiaXRjb2luLm9yZzAw
    MDAwMDAwMDAwMDAwMDAwYjg4ZmJiZGQyMTI1NmVjYTM5ZmZlYjJkMmZmNmE5MTg1
    NGExOGUwYjczMTI3OGQvFIAAAAAAFQARcGthLWFkZHJlc3NAZ251cGcub3JncGV0
    ZUBwZXRlcnRvZC5vcmcACgkQwIXyHOf0udy4Wgf/c8Rjflb7QLzSpHOi2GcUdtX5
    OnJFCQ/bB9J56lKMS6vWcB5WRRtyD2rzyNDp3rvmn7FmgnIJaxw/MPx+1PmFl5ok
    apblmjvbPoAIkRkd+Nhp8cV9UsZel0JcNu2tGOLyWDl/fzvTkZ4NpLbvAyyX+jHm
    YyDwKgOySm6wgWybC1j2fPf6jda3cV0gtluDx/kS2wP1jZNH73ncWvBYaSXbrBt9
    m2jyRwAoWVGU7SEguqm/+refI2ytt7QLsfkyQmtYDUkNR54NzyiEEaLlFro8Uioi
    WfIab62wjohWwt4/u3+mhpFgaqrgHFkVtYdh1fI+PXrUF52HNX3kdHXloorsxw==
    =hN73
    -----END PGP SIGNATURE-----

**Comment by petertodd** [comment]
CC: @scisci

**Comment by maxtaco** [comment]
Thanks @petertodd  I was debugging a related issue the other day.  I think this is a case of a bad error message and we just don't have a PGP key for this user.

[This call](https://keybase.io/_/api/1.0/key/fetch.json?pgp_key_ids=c085f21ce7f4b9dc&ops=4) is failing.

**Comment by petertodd** [comment]
Yup, I just refreshed my key and it works now; just need a better error message then.

**Comment by Brianetta** [comment]
Was about to log this, but found it here already. The CLI tool uses a suitable error message:
`▶ ERROR Check sig error: openpgp: signature made by unknown entity`
